main/src/ca/uwaterloo/flix/tools/pkg/MavenPackageManager.scala

```diff
@@ -15,25 +15,45 @@
  */
 package ca.uwaterloo.flix.tools.pkg
 
+import ca.uwaterloo.flix.tools.pkg.Dependency.MavenDependency
 import ca.uwaterloo.flix.util.Result
+import ca.uwaterloo.flix.util.Result.{Err, ToOk}
 
 import java.io.PrintStream
 import coursier._
 
 object MavenPackageManager {
 
-  def installDeps(manifest: Manifest)(implicit out: PrintStream): Result[Unit, ManifestError] = {
-
-
-    val resolution = Resolve()
-      .addDependencies(dep"org.tpolecat:doobie-core_2.12:0.6.0")
-      .run()
-    println(resolution)
-
-    ???
-
+  /**
+    * Installs all MavenDependencies for a Manifest,
+    * including transitive dependencies using coursier.
+    */
+  def installDeps(manifest: Manifest)(implicit out: PrintStream): Result[Unit, PackageError] = {
```

Comment:
Do you want to return a list of installed dependencies here, like in FlixPackageManager?

Reply:
Yes, I think for both that would be useful.